crates/pallet-domains/src/staking_epoch.rs

```diff
@@ -141,6 +150,10 @@ pub(crate) fn operator_take_reward_tax_and_stake<T: Config>(
             })?;
         }
 
+        if !to_treasury.is_zero() {
+            mint_into_treasury::<T>(to_treasury).ok_or(TransitionError::MintBalance)?;
```

Comment:
can't we just make this function handle the edge case instead. Its easy to miss otherwise

Reply:
Done here https://github.com/autonomys/subspace/pull/3285